### DISCUSSION TITLE
CS-PMP%40UWM/

### DISCUSSION THREAD

**Discussion by giscus** [opencsapp/opencsapp.github.io#88] [Announcements]
# CS-PMP%40UWM/

https://csmsapp.github.io/CS-PMP%40UWM/

**Comment by yuhengcai1** [comment]
PMP过于良心，本校学生都想申请了，怎么wisc这研究生学费收这么低

**Reply by xichenpan** [reply]
求大佬给pr一下项目描述haha~

**Reply by yuhengcai1** [reply]
学费大概一年只要18个学分的钱，连本科生一年都要24个学分的钱，学费极其便宜，大概一年学费只要3万出头，如果ta或者ra的话更加便宜，听朋友说有清华大佬放弃cmu的mscs来pmp的，估计就是图一个省钱

**Comment by Allyyi** [comment]
PMP是授课型，不可以通过TA/RA接受tuition remission，但是学费已经很划算了
